src/system/irrigation.cpp

```diff
@@ -5,6 +5,8 @@
 #include "utils/logger.h"
 #include "relays/builder.h"
 
+using namespace WiFiConnection;
```

Comment:
This using namespace has no effect, you're already referencing WiFiInitialize explicitly 
```c
WiFiConnection::WifiInitialize();
```
We use using namespace only when we want to avoid writing the namespace before the calls.